internal/promoter/image/sign.go

```diff
@@ -96,6 +103,7 @@ func (di *DefaultPromoterImplementation) CopySignatures(
 ) error {
 	// Cycle all signedEdges to copy them
 	logrus.Infof("Precopying %d previous signatures", len(signedEdges))
+	rtripper := ratelimit.NewRoundTripper(rateLimitEvents)
```

Comment:
This object is thread safe, maybe a global until we can more easily plumb through a single copy?

Reply:
I was just moving it to a global :) And also added the rate limiter to the promotion crane calls.